README.md

```diff
@@ -1,159 +1,173 @@
+* Windows Terminal Launch: [Build 2019 Session](https://www.youtube.com/watch?v=KMudkRcwjCw)
+* Run As Radio: [Show 645 - Windows Terminal with Richard Turner](http://www.runasradio.com/Shows/Show/645)
+* Azure Devops Podcast: [Episode 54 - Kayla Cinnamon and Rich Turner on DevOps on the Windows Terminal](http://azuredevopspodcast.clear-measure.com/kayla-cinnamon-and-rich-turner-on-devops-on-the-windows-terminal-team-episode-54)
+---
 
-The latest release of Windows Terminal can be downloaded from [this repo's Releases](https://github.com/microsoft/terminal/releases) page or [from the Microsoft Store](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
+## FAQ
 
-## I built and ran the new Terminal, but I just get a blank window app!
+## I built and ran the new Terminal, but I just see a blank window app
 
-Make sure you are building for your computer's architecture. If your box has a 64-bit Windows, change your Solution Platform to x64.  
-To check your OS architecture go to Settings -> System -> About (or Win+X -> System) and under `Device specifications` check for the  `System type`.
+Make sure your Terminal us built for your PC's processor architecture:
 
-## I built and ran the new Terminal, but it looks just like the old console! What gives?
+For example, if your PC runs 64-bit Windows, set your Solution Platform to x64: In Visual Studio, go to Settings -> System -> About (or Win+X -> System) and under `Device specifications` check for the  `System type`.
 
-Firstly, make sure you're building & deploying `CascadiaPackage` in Visual Studio, _NOT_ `Host.EXE`. `OpenConsole.exe` is just `conhost.exe`, the same old console you know and love. `opencon.cmd` will launch `openconsole.exe`, and unfortunately, `openterm.cmd` is currently broken.
+## I built and ran the new Terminal, but it looks just like the old console
 
-Secondly, try pressing <kbd>Ctrl</kbd> + <kbd>T</kbd>. The tabs are hidden when you only have one tab by default. In the future, the UI will be dramatically different, but for now, the defaults are _supposed_ to look like the console defaults.
+Make sure you're building & running the `CascadiaPackage` project in Visual Studio.
 
-## I tried running WindowsTerminal.exe and it crashes!
+## I tried running WindowsTerminal.exe and it crashes
 
-* Don't try to run it unpackaged. Make sure to build & deploy `CascadiaPackage` from Visual Studio, and run the Windows Terminal (Dev Build) app.
-* Make sure you're on the right version of Windows. You'll need to be on Insider's builds, or wait for the 1903 release, as the Windows Terminal **REQUIRES** features from the latest Windows release.
+* Don't try to run `WindowsTerminal.exe` directly. Make sure to build & deploy `CascadiaPackage` from Visual Studio, and run the Windows Terminal (Dev Build) app.
+* Make sure you're running Windows 10 1903 or later - the Windows Terminal **REQUIRES** features from the latest Windows release.
```

Comment:
"features from the latest Windows release" is about to become dated language with our next release, isn't it? Maybe we should just be specific about 1903+